src/frontend/StereoFrame.cpp

```diff
@@ -1209,8 +1209,8 @@ std::pair<StatusKeypointCV, double> StereoFrame::findMatchingKeypointRectified(
 
   // Refine keypoint with subpixel accuracy.
   if (sparse_stereo_params_.subpixel_refinement_) {
-    static const cv::TermCriteria criteria(CV_TERMCRIT_EPS + CV_TERMCRIT_ITER,
-                                           40, 0.001);
+    static const cv::TermCriteria criteria(
+        cv::TermCriteria::EPS + cv::TermCriteria::COUNT, 40, 0.001);
```

Comment:
no preprocessor checks here?

Reply:
Those enums are already in OpenCV3 so no preprocessor checks needed.